src/fire/sendMessageFire.js

```diff
@@ -1,8 +1,45 @@
-//import fire from './Fire.js';
+import {fire, db} from './Fire.js';
 
 
 function sendMessageFirebase(event) {
-  // nothing yet
+  // Proccess incoming data from the form
+  event.preventDefault();
+  const data = new FormData(event.target);
+  var messageTo = (data.get('messageTo'));
+  var messageBody = (data.get('messageBody'));
+  console.log("Sending a message into db...")
+  console.log("messageTo: ", messageTo);
+  console.log("messageBody: ", messageBody);
```

Comment:
Once we are able to display messages, remember to remove these. Fine for now.

Reply:
Noted.

Reply:
Will create an issue to clean up debug prints before the "final" release.